### PR TITLE
upd(micro): `2.0.8` -> `2.0.10`

### PR DESCRIPTION
**Pull request by cat-master21**

### CODE REVIEW

File: packages/micro/micro.pacscript [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
 
 build() {
-    chmod +x micro
+    chmod +x "./micro"
 }
 
 install() {
-    sudo mkdir -p /usr/src/pacstall/micro/usr/bin/
-    sudo cp micro /usr/src/pacstall/micro/usr/bin
+    sudo cp micro "/usr/src/pacstall/micro/usr/bin"
```
Comment (Henryws) [inline packages/micro/micro.pacscript:20]:
```suggestion
    sudo install -Dm644 micro "/usr/src/pacstall/micro/usr/bin/micro"
```

File: packages/micro/micro.pacscript [hunk 2]
```diff
@@ -1,22 +1,25 @@
...
+    tar -czf "micro.1.tar.gz" "micro.1"
+    sudo cp micro.1.tar.gz "/usr/src/pacstall/micro/usr/share/man/man1"
+    sudo install -Dm644 "./micro.svg" "/usr/src/pacstall/micro/usr/share/pixmaps/micro.svg"
+    sudo cp "micro.desktop" "/usr/src/pacstall/micro/usr/share/applications"
```
Comment (Henryws) [inline packages/micro/micro.pacscript:24]:
```suggestion
    sudo install -Dm644 micro.desktop "/usr/src/pacstall/micro/usr/share/applications/micro.desktop"
```

File: packages/micro/micro.pacscript [hunk 3]
```diff
@@ -1,22 +1,25 @@
...
-    chmod +x micro
+    chmod +x "./micro"
 }
 
 install() {
-    sudo mkdir -p /usr/src/pacstall/micro/usr/bin/
-    sudo cp micro /usr/src/pacstall/micro/usr/bin
+    sudo cp micro "/usr/src/pacstall/micro/usr/bin"
+    tar -czf "micro.1.tar.gz" "micro.1"
+    sudo cp micro.1.tar.gz "/usr/src/pacstall/micro/usr/share/man/man1"
```
Comment (Henryws) [inline packages/micro/micro.pacscript:22]:
```suggestion
    sudo install -Dm644 micro.1.tar.gz "/usr/src/pacstall/micro/usr/share/man/man1/micro.1.tar.gz"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/micro/micro.pacscript [hunk 4]
```diff
@@ -1,22 +1,25 @@
...
-build_depends="build-essential golang"
+url="https://github.com/zyedidia/micro/releases/download/v${version}/micro-${version}-linux64.tar.gz"
+build_depends="build-essential"
 gives="micro"
 breaks="micro"
-hash="7a705a60fe5da8fdfbb1423b560ae120a59af963ec73e3548ef97a6a688cd7cf"
+hash="98797d745aec85b36a9cedc87ce7656adcd8cc94c909a07edeb6e61538418bed"
 prepare() {
-    true
+    sudo mkdir -p "/usr/src/pacstall/micro/usr/bin" "/usr/src/pacstall/micro/usr/share/man/man1" "/usr/src/pacstall/micro/usr/share/applications"
```
Comment (Henryws) [inline packages/micro/micro.pacscript:12]:
```suggestion

```

**Review by Henryws — CHANGES_REQUESTED** [review]
Remove this line, because it is useless now that `install` commands are used

**Review by Henryws — APPROVED** [review]

File: packages/micro/micro.pacscript [hunk 5]
```diff
@@ -1,22 +1,25 @@
...
 }
 
 install() {
-    sudo mkdir -p /usr/src/pacstall/micro/usr/bin/
-    sudo cp micro /usr/src/pacstall/micro/usr/bin
+    sudo install -Dm644 micro "/usr/src/pacstall/micro/usr/bin/micro"
+    tar -czf "micro.1.tar.gz" "micro.1"
+    sudo install -Dm644 micro.1.tar.gz "/usr/src/pacstall/micro/usr/share/man/man1/micro.1.tar.gz"
+    sudo install -Dm644 "./micro.svg" "/usr/src/pacstall/micro/usr/share/pixmaps/micro.svg"
+    sudo install -Dm644 micro.desktop "/usr/src/pacstall/micro/usr/share/applications/micro.desktop"
```
Comment (Henryws) [inline packages/micro/micro.pacscript:24]:
```suggestion
    sudo install -Dm644 micro.desktop "/usr/src/pacstall/micro/usr/share/applications/micro.desktop"
    sudo chmod +x "/usr/src/pacstall/micro/usr/bin/micro"
```

**Review by Henryws — CHANGES_REQUESTED** [review]
Oops forgot to chmod it

**Review by Henryws — APPROVED** [review]

**Comment by wizard-28** [comment]
@cat-master21 Would you like to make yourself the maintainer of this package?

@edward-riley Are you okay with transferring the responsibility to @cat-master21?

**Pull request merged**